trieste/models/interfaces.py

```diff
@@ -224,7 +224,7 @@ def conditional_predict_joint(
         :param additional_data: Dataset with query_points with shape [..., N, D] and observations
                  with shape [..., N, L]
         :return: The mean and covariance of the joint marginal distribution at each batch of points
```

Comment:
Also, what is the "joint marginal distribution"? is this the joint distribution over all *f*s? So should this be
```suggestion
        :return: The mean and covariance of the joint distribution at each batch of points
```
?